shared/categories.ts

```diff
@@ -0,0 +1,1463 @@
+	'N2:BOJ',
+	'N2:ECB',
+	'N2:FED',
+	'N2:IMF',
+	'N2:RBA',
+	'N2:SEN',
+	'N2:SNB',
+	'N2:WTO',
+	'N2:AFRU',
+	'N2:APEC',
+	'N2:BMXO',
+	'N2:CBBR',
+	'N2:CBNG',
+	'N2:CBRT',
+	'N2:CBRU',
+	'N2:HNBK',
+	'N2:HREP',
+	'N2:NBPL',
+	'N2:OECD',
+	'N2:OPEC',
+	'N2:PBOC',
+	'N2:RBIN',
+	'N2:RBNZ',
+	'N2:SARB',
+	'N2:USDA',
+	'N2:WBNK',
+	'N2:NORBK',
+	'N2:RIKBK',
+	'N2:ASEAN1',
+	// Sectors
+	'N2:LUX',
+	'N2:MIN',
+	'N2:TMT',
+	'N2:AERO',
+	'N2:AGRC',
+	'N2:AIRL',
+	'N2:AIRP',
+	'N2:ALUM',
+	'N2:AMED',
+	'N2:APPA',
+	'N2:APRE',
+	'N2:AUTO',
+	'N2:BCST',
+	'N2:BEER',
+	'N2:BEVS',
+	'N2:BGDS',
+	'N2:BISV',
+	'N2:BMAT',
+	'N2:BNKS',
+	'N2:BSUP',
+	'N2:BSVC',
+	'N2:CARM',
+	'N2:CCOS',
+	'N2:CELE',
+	'N2:CHEM',
+	'N2:CHIP',
+	'N2:CIGS',
+	'N2:CMSS',
+	'N2:COAM',
+	'N2:COEN',
+	'N2:COFS',
+	'N2:COMA',
+	'N2:COMC',
+	'N2:COMS',
+	'N2:CONG',
+	'N2:CONS',
+	'N2:CYCP',
+	'N2:CYCS',
+	'N2:DEPT',
+	'N2:DISC',
+	'N2:DRIL',
+	'N2:ELCO',
+	'N2:ELEU',
+	'N2:ENER',
+	'N2:ENTS',
+	'N2:ENVS',
+	'N2:EXCA',
+	'N2:FARM',
+	'N2:FDRT',
+	'N2:FINS',
+	'N2:FOBE',
+	'N2:FOTB',
+	'N2:GASU',
+	'N2:HARW',
+	'N2:HECA',
+	'N2:HELE',
+	'N2:HFAC',
+	'N2:HMAC',
+	'N2:HMOS',
+	'N2:HOME',
+	'N2:HPAS',
+	'N2:HPRD',
+	'N2:HRSC',
+	'N2:INDG',
+	'N2:INDM',
+	'N2:INDS',
+	'N2:INSB',
+	'N2:INSR',
+	'N2:INVM',
+	'N2:INVT',
+	'N2:ISER',
+	'N2:ISLF',
+	'N2:ITSE',
+	'N2:LEIS',
+	'N2:LINS',
+	'N2:LIQS',
+	'N2:MACH',
+	'N2:MDIA',
+	'N2:MEDQ',
+	'N2:MEMI',
+	'N2:MINE',
+	'N2:MINS',
+	'N2:MRCH',
+	'N2:MTAL',
+	'N2:NCYC',
+	'N2:OFCE',
+	'N2:OGTR',
+	'N2:OILG',
+	'N2:OILI',
+	'N2:OILQ',
+	'N2:PACK',
+	'N2:PAPR',
+	'N2:PHAR',
+	'N2:PHMR',
+	'N2:PHON',
+	'N2:PINS',
+	'N2:PLAY',
+	'N2:PREC',
+	'N2:PRIN',
+	'N2:PSVC',
+	'N2:RDIY',
+	'N2:RDRL',
+	'N2:RDRU',
+	'N2:READ',
+	'N2:REAL',
+	'N2:REFI',
+	'N2:REIC',
+	'N2:REID',
+	'N2:REIR',
+	'N2:REIT',
+	'N2:RELX',
+	'N2:RENE',
+	'N2:RENF',
+	'N2:RENQ',
+	'N2:REOP',
+	'N2:RESV',
+	'N2:RETA',
+	'N2:RETE',
+	'N2:RETS',
+	'N2:RFOD',
+	'N2:RGEN',
+	'N2:RHOM',
+	'N2:RINS',
+	'N2:RRLF',
+	'N2:RRLP',
+	'N2:RUBB',
+	'N2:SBIZ',
+	'N2:SEMI',
+	'N2:SHOE',
+	'N2:SHOP',
+	'N2:SHOW',
+	'N2:SOFW',
+	'N2:STAY',
+	'N2:SWIT',
+	'N2:TCOM',
+	'N2:TECH',
+	'N2:TEEQ',
+	'N2:TRAD',
+	'N2:TRAN',
+	'N2:TREE',
+	'N2:TXTL',
+	'N2:UTIL',
+	'N2:UTIM',
+	'N2:WATR',
+	'N2:WEAR',
+	'N2:WLES',
+	'N2:WLTH',
+	'N2:WOOD',
+	'N2:ADVMK',
+	'N2:APRET',
+	'N2:AUTPT',
+	'N2:CASGM',
+	'N2:CHMSP',
+	'N2:CRDBX',
+	'N2:EXPRO',
+	'N2:FODPR',
+	'N2:HMFRN',
+	'N2:HRDWR',
+	'N2:HSEPR',
+	'N2:INVBR',
+	'N2:IRNST',
+	'N2:NPCNT',
+	'N2:PRTSV',
+	'N2:PSNPR',
+	'N2:REALE',
+	'N2:RECPR',
+	'N2:RESBR',
+	'N2:SEAFT',
+	'N2:SFTBV',
+	'N2:VHRET',
+	'N2:AIRFRT',
+	'N2:CHEMDV',
+	'N2:ENQSVC',
+	'N2:HLTHSE',
+	'N2:HSEBLD',
+	'N2:HSEGDS',
+	'N2:HSEHLD',
+	'N2:INVBIS',
+	'N2:INVDIV',
+	'N2:SHOPAL',
+	'N2:SPCRET',
+	'N2:NO EQUIVALENT',
+];
+export const AllBusiness = [
+	'apCat:f',
+    'paCat:FFF',
+    'paCat:GXX',
+    ...businessRelatedNewsCodes,
+    ...businessRelatedTopicCodes
+];
+export const Soccer = [
+    'N2:SOCC',
```

Comment:
there's a duplication here -- is it just duplicated or are there meant to be two variations?

Reply:
Oh thanks, good spot